internal/errs/errs.go

```diff
@@ -104,6 +131,22 @@ func Wrap(wrapTarget error, message string, args ...interface{}) *WrapperError {
 	return newError(msg, wrapTarget)
 }
 
+func NewUserFacingError(message string, args ...interface{}) *userFacingError {
+	return &userFacingError{
+		nil,
+		fmt.Sprintf(message, args...),
+		nil,
+	}
+}
+
+func WrapUserFacingError(wrapTarget error, message string, args ...interface{}) *userFacingError {
```

Comment:
Seeing as user facing errors should always be localized we should just take a single message arg, no further args. Because splicing the message together is the job of the localization lib.